cert-manager-multi-arch.md

```diff
@@ -0,0 +1,728 @@
+  - [Certificate Management:Example](#certificate-managementexample)
+    - [Create Self-signed CA Issuer](#create-self-signed-ca-issuer)
+    - [Issue Certificate](#issue-certificate)
+    - [Change ingress controller's default certificate](#change-ingress-controllers-default-certificate)
+    - [Replace the CA bundle certificate](#replace-the-ca-bundle-certificate)
+    - [See it in action](#see-it-in-action)
+  - [Wrap Up](#wrap-up)
+  - [Useful links](#useful-links)
+
+
+# Manage certificates on OpenShift running on IBM Z®, IBM Power® and ARM64 architectures.
+
+
+## Introduction
+
+Kubernetes thrives on secure communication, but manually managing SSL/TLS certificates is a tangled mess and error-prone process, leading to outages, security vulnerabilities, and operational overhead.
+
+
+The [cert-manager](https://docs.openshift.com/container-platform/4.14/security/cert_manager_operator/index.html) Operator for Red Hat OpenShift provides a secure and efficient solution for TLS certificate management in OpenShift Container Platform clusters, by introducing certificates and certificate issuers as primary resources in the Kubernetes API.  This _**'certificates as a service'**_ model seamlessly integrates with external certificate authorities, automating the entire certificate lifecycle, from provisioning to renewal, ensuring validity and timely updates.
+
+Now, with the latest update, the cert-manager Operator for Red Hat OpenShift 1.13.0 has undergone an expansion in its scope. Formerly confined to supporting solely on `AMD64` architecture, it now includes extended support for managing certificates on OpenShift across multiple architectures, including IBM Z® (`s390x`), IBM Power® (`ppc64le`), and `ARM64`. 
+
+<p align="center">
+  <img src="assets/002-multi-arch+openshift+cm.png" alt>
+  <em>Figure 1: cert-manager Operator for Red Hat OpenShift's robust support for multiple architectures.</em>
+</p>
+
+
+In this article, I'll briefly explore multi-arch container images before delving into the process of setting up an OpenShift cluster on IBM Power®. I'll then proceed to the cert-manager Operator installation on Power VS and demonstrate its use in certificate management. 
+
+It's worth noting that comparable steps can be applied to other supported architectures. I've provided links to relevant documents for further reference.
+
+So, let's begin!
+
+
+
+## Multi-architecture container image
+
+### What is it?
+Multi-architecture container images represent a containerization strategy where a single container image is crafted to run seamlessly across various CPU architectures (e.g. x86, ARM, s390x, RISC-V) and sometimes operating systems (e.g., Linux, Windows). A common tag is used for the image, but the container runtime automatically selects the appropriate variant based on the target platform where you want to run the container, using manifests  that reference individual image variants for different platforms.
```

Comment:
`single container image`, probably replace with `reference to a unified container image`?
As technically each arch has it's own image but all of them just share a common image reference.